ubi_manifest/worker/tasks/depsolver/rpm_depsolver.py

```diff
@@ -256,21 +286,22 @@ def run(self) -> None:
             # we are finished if _unsolved is empty
             if not self._unsolved:
                 break
-
+            # resolve any remaining file dependencies
+            resolved = self.resolve_files(pulp_repos, merged_blacklist)
```

Comment:
I a bit worried with performance of this call. I guess only testing on real data can prove that.

Reply:
Yeah. And I don't think there's anything we can do to improve queries on this end anyway.